sql/importer: TestRandomParquetExports flakes

```console
./dev test ./pkg/sql/importer/ -f TestRandomParquetExports --ignore-cache --stress --cpus 16 --timeout 10m
--- FAIL: TestRandomParquetExports (2.39s)
    test_log_scope.go:167: test logs captured to: /tmp/cockroach/_tmp/73dfce30b9a5630b1b4dabed3c94b32c/logTestRandomParquetExports3282656100
    test_log_scope.go:81: use -show-logs to present logs inline
    exportparquet_test.go:257: exporting as parquet from random table with schema: 
         [["t ab le1" CREATE TABLE public."t ab le1" (
                rowid INT8 NOT VISIBLE NOT NULL DEFAULT unique_rowid(),
                CONSTRAINT "t ab le1_pkey" PRIMARY KEY (rowid ASC)
        )]]
    exportparquet_test.go:104: 
                Error Trace:    github.com/cockroachdb/cockroach/pkg/sql/importer_test/pkg/sql/importer/exportparquet_test.go:104
                                                        github.com/cockroachdb/cockroach/pkg/sql/importer_test/pkg/sql/importer/exportparquet_test.go:267
                Error:          Received unexpected error:
                                strconv.Atoi: parsing "": invalid syntax
                Test:           TestRandomParquetExports
    panic.go:522: -- test log scope end --
FAIL
I230717 13:48:59.298028 1 (gostd) testmain.go:261  [-] 1  Test //pkg/sql/importer:importer_test exited with error code 1


ERROR: exit status 1

40 runs completed, 1 failures, over 11s
```

Also seen in CI: https://teamcity.cockroachdb.com/viewLog.html?buildId=10919135&buildTypeId=Cockroach_BazelEssentialCi. 

Jira issue: CRDB-29760

--- a/pkg/sql/importer/exportparquet_test.go
+++ b/pkg/sql/importer/exportparquet_test.go
@@ -100,6 +100,18 @@ func validateParquetFile(
 		validationStmt)
 	require.NoError(t, err)
 
+	// It's possible to have a set of rows where each row has no columns.
+	// Ex.    CREATE TABLE public.tabl͛e1 (
+	//                rowid INT8 NOT VISIBLE NOT NULL DEFAULT unique_rowid(),
+	//                CONSTRAINT tabl͛e1_pkey PRIMARY KEY (rowid ASC)
+	//        )
+	// In this case, the parquet file will simply contain no datums, but
+	// the SELECT above returns a slice of empty slices [[], [], []...].
+	// Thus, we override the array of test datums as an empty slice [].
+	if len(test.datums) > 0 && len(test.datums[0]) == 0 {
+		test.datums = make([]tree.Datums, 0)
+	}
+
 	meta, readDatums, err := crlparquet.ReadFile(paths[0])
 	require.NoError(t, err)
 

--- a/pkg/util/parquet/testutils.go
+++ b/pkg/util/parquet/testutils.go
@@ -499,6 +499,13 @@ func serializeIntArray[I int | int32 | uint32](ints []I) string {
 // 32]" to an array of ints.
 func deserializeIntArray(s string) ([]int, error) {
 	vals := strings.Split(strings.Trim(s, "[]"), " ")
+
+	// If there are no values, strings.Split returns an array of length 1
+	// containing the empty string.
+	if len(vals) == 0 || (len(vals) == 1 && vals[0] == "") {
+		return []int{}, nil
+	}
+
 	result := make([]int, 0, len(vals))
 	for _, val := range vals {
 		intVal, err := strconv.Atoi(val)

